beacon-chain/sync/rpc.go

```diff
@@ -25,29 +26,29 @@ type rpcHandler func(context.Context, interface{}, libp2pcore.Stream) error
 // registerRPCHandlers for p2p RPC.
 func (r *RegularSync) registerRPCHandlers() {
 	r.registerRPC(
-		"/eth2/beacon_chain/req/hello/1",
-		&pb.Hello{},
-		r.helloRPCHandler,
+		"/eth2/beacon_chain/req/status/1",
+		&pb.Status{},
+		r.statusRPCHandler,
 	)
 	r.registerRPC(
 		"/eth2/beacon_chain/req/goodbye/1",
-		&pb.Goodbye{},
+		new(uint64),
```

Comment:
We shouldn't need to do this anymore, just pass in uint64

Reply:
That wont work, uint64 alone is not regarded as a valid interface{}, you have to make it to an expression to make it to pass